src/iamclient/iamclient.cpp

```diff
@@ -200,7 +198,7 @@ Error IAMClient::SetupChannel()
         mCurrentPort = cSecurePort;
 
 #ifndef CONFIG_ZTEST
-        if (err = mTLSChannel.Init(*mCertHandler, *mCertLoader, *channel); !err.IsNone()) {
+        if (err = mTLSChannel.Init("iam", *mCertHandler, *mCertLoader, *channel); !err.IsNone()) {
```

Comment:
consider adding a const to hold "iam" value

Reply:
used only once for log purpose